app/src/main/java/org/connectbot/service/ConnectionNotifier.java

```diff
@@ -43,6 +44,7 @@
  */
 public abstract class ConnectionNotifier {
 	private static final int ONLINE_NOTIFICATION = 1;
+	private static final int ONLINE_NOTIFICATION_DISCONNECT = 1;
```

Comment:
Ah sorry, this snuck in there when I was playing with adding the disconnect button. Removed.